Resources/config/lists/comments.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" ?>
+<list xmlns="http://schemas.sulu.io/list-builder/list">
+    <key>comments</key>
+
+    <joins name="creator">
+            <join>
+                <entity-name>%sulu.model.user.class%</entity-name>
+                <field-name>%sulu.model.comment.class%.creator</field-name>
+            </join>
+            <join>
```

Comment:
I personally prefer have a empty line before a multiline opening tag for readability 🙈